justfile

```diff
@@ -181,6 +181,24 @@ test-int: clean-test install-sqlx
         fi
     fi
 
+# Generate code coverage report
+coverage *ARGS="--open": clean start (cargo-install "cargo-llvm-cov")
```

Comment:
please keep these in the same place in the just file - makes it much easier to review the changes vs cleanups